### PR TITLE
feat: add button in tree view

### PR DESCRIPTION
**Pull request by IvanJobs**
to add cicd button in vscode-extension.

the tree view looks like:
image

to finish work item: https://dev.azure.com/msazure/Microsoft%20Teams%20Extensibility/_workitems/edit/10430612

### CODE REVIEW

**Comment by 1openwindow** [comment]
@IvanJobs we are going to A/B test the tree view. so what does this looks like in new Tree View UI test.

**Comment by IvanJobs** [comment]
> so what does this looks like in new Tree View UI test.

We may discuss offline about this.

File: packages/vscode-extension/src/resources/Strings.json [hunk 1]
```diff
@@ -41,6 +41,8 @@
       "deployDescription": "Deploy your Teams app to the provisioned resources in Azure",
       "publishTitle": "Publish to Teams",
       "publishDescription": "Publish to Teams",
+      "cicdGuideTitle": "Go To CI/CD Guide",
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/src/resources/Strings.json:44]: Better align the format with other items, for example, the title for deploy  is 'Deploy to the Cloud', both 'to ' and 'the' start with lowercase

Comment (IvanJobs) [reply packages/vscode-extension/src/resources/Strings.json:44]: Agree, change it soon.

Comment (IvanJobs) [reply packages/vscode-extension/src/resources/Strings.json]: According to UX's suggestion, update it to `CI/CD guide`.

**Review by 1openwindow — APPROVED** [review]

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Pull request merged**